src/components/toast/toast.spec.js

```diff
@@ -82,6 +70,17 @@ describe("Toast", () => {
       });
     });
 
+    it("does not renders close icon", () => {
```

Comment:
```suggestion
    it("does not render close icon", () => {
```

Reply:
looks like this was a 6 years old typo :) good catch!